ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/resources/arjunacore/CommitMarkableResourceRecord.java

```diff
@@ -112,17 +113,13 @@
 	private static CommitMarkableResourceRecordRecoveryModule commitMarkableResourceRecoveryModule;
 	private static final JTAEnvironmentBean jtaEnvironmentBean = BeanPopulator
 			.getDefaultInstance(JTAEnvironmentBean.class);
-	private static final Map<String, String> commitMarkableResourceTableNameMap = jtaEnvironmentBean
-			.getCommitMarkableResourceTableNameMap();
 	private static final String defaultTableName = jtaEnvironmentBean
 			.getDefaultCommitMarkableTableName();
 	private boolean isPerformImmediateCleanupOfBranches = jtaEnvironmentBean
```

Comment:
Is it fine to leave other configuration options for CMR as `static` here?

Reply:
The WildFly CMR service does not touch these other properties.

Reply:
I see, +1

In general I think it would be good to handle all the CMR properties in the same way and not differently either they are used in WFLY or not. But it's my personal standpoint and I do not insist on it.

Reply:
Caching the per CM resource properties (batch-size, immediate-cleanup, table name) will not cause the data integrity issue that this PR is fixing.

However, you are correct in pointing out that these properties can be set on a per CM resource basis but narayana assumes that they are global properties. I will raise a separate issue to fix this.

Reply:
I'm fine here. The issue could be good while I agree that these ones are the problematic causing the integrity issue. I meant this like a note and resolving this.